File: sum.c
#include<stdio.h>
int main(){
    int number,i,sum=0;
    printf("enter n mumbers");
    scanf("%d",&number);
    for(i=1;i<=number;i++){
        sum=sum+i;
    }
    printf("Sum = %d\n", sum);

    return 0;
}

File: sum.java

import java.util.Scanner;

public class sum {
    public static void main(String[] args) {
        Scanner sc= new Scanner(System.in);
        int[] arr=new int[5];
        int sum=0,i;
        for ( i = 0; i < 5; i++) {
            System.out.println("Enter the number in array:");
            arr[i]=sc.nextInt();
          
        }
        for(i=0;i<arr.length;i++){
            // System.out.println("sum of all the array are");
        sum = sum + arr[i];
         
        }
        System.out.println("sum of all the array is:");
       System.out.println(sum);
    }
}
